Olwa pr (#36)
hms-networks/flexy-thingworx-connector: README.md

   1. [Installation](#installation)
   2. [Configuration](#configuration)
      1. [Thingworx Full URL](#thingworx-full-url)
      2. [App Key](#app-key)
      3. [Log Level](#log-level)

>       4. [FTP User](#ftp-user)

Reviewer: ```suggestion
      4. [FTP Username](#ftp-username)
```

After the Flexy Java application component has been installed and started, the configuration should be modified to contain the proper values for AppKey and Thingworx Full URL for your Thingworx instance.

![Configuration Example](https://github.com/hms-networks/flexy-thingworx-connector/blob/local-write-back/images/ExampleConfig.PNG?raw=true)

#### Thingworx Full URL
This parameter in the config file should be changed to the full URL for the target Thingworx REST API. Users should pay close attention to the scheme designated in the URL. Specifying the scheme "https" will ensure all traffic to the server is encrypted over TLS. For example, setting the ThingworxFullUrl parameter to "https://example.com:8443/Thingworx/Things/ConnectorHost/Services/TakeInfo" will create a connection to host "example.com" on port 8443 using TLS/SSL. It's possible to specify the IP address of the Thingworx instance instead of a domain name. This configuration will assume "http" scheme and communication will not be encrypted. 

#### App Key

> This should match the app key for your Thingworx instance.

Reviewer: ```suggestion
This should match the app key for the Thingworx instance.
```

#### Log Level
The Thingworx connector uses the HMS Solution Center logging library for application logging to the Ewon Flexy's realtime logs. See [Log Level](https://github.com/hms-networks/sc-flexy-logger-lib#log-level) for more information.

#### FTP User
The username for accessing Ewon Flexy via FTP. This user account must be configured for non UTC time zones to be used on the Flexy.

#### FTP Password
The password for accessing Ewon Flexy via FTP. This user account must be configured for non UTC time zones to be used on the Flexy.

> 

Reviewer: If the intention is that the user will follow the readme for installation instructions we should provide more detail about the FTP user such as what permissions it needs. This would probably be another section in the readme and could be addressed in another PR.

Author: I will include a new section on setting up the FTP user in the next PR.

#### Queue Enable String History

> Boolean flag indicating if string history data should be retrieved from the queue. String history requires an additional EBD call in the underlying queue library, and may take extra processing time, especially in installations with large string tag counts.

Reviewer: ```suggestion
Boolean flag indicating if string history data should be retrieved from the queue. String history requires an additional EBD call in the underlying queue library, and will take extra processing time, especially in installations with large string tag counts.
```

#### Queue Data Poll Size

> The data poll size in minutes of each data queue poll. Changing this will modify the amount of data checked during each poll interval. If not used, the default value will be 1.

Reviewer: same thing with referencing exact values as below. If you really want them to know what that value is maybe reference the default variable name.

#### Queue Data Poll Interval

> The data poll interval in milliseconds to poll the historical data queue. If not used, the default value will be 30000.

Reviewer: you might not want to directly reference the default value in the readme. This will just lead to a out of date doc if you ever change that value.
```suggestion
Optional parameter to override the default data poll interval (in milliseconds) to poll the historical data queue.
```
